vue/components/ui/molecules/lineup/lineup.vue

```diff
@@ -85,56 +86,50 @@
     text-align: left;
 }
 
-.lineup > .lineup-item {
+.lineup .lineup-item {

```

Comment:
Why the change?

Reply:
Redid the style, no particular reason.

Reply:
Revert it then, let's keep following the principle of least action. Unless something is a clear improvement we'd be better to avoid changing it.